src/style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --color-teal: #00acdc;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+}
+
+.card__container {
+  width: 166px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__caption {
+  margin-top: 40px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Move styles for stars to a separate file
![image](https://user-images.githubusercontent.com/96409563/219943061-ab54d3e1-6101-4bf0-90d1-1236fa13d3b6.png)